app/javascript/components/patient/Patient.js

```diff
@@ -17,7 +17,7 @@ class Patient extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      expanded: props.edit_mode || !props.collapse,
+      expanded: props.hidePreviousButton || !props.collapse,
```

Comment:
this should stay `edit_mode` (see above comment)

Reply:
fixed in https://github.com/SaraAlert/SaraAlert/pull/975/commits/6620f09e6346a83c99080db3e9e12eef8a1ee5f4